.github/workflows/benchmark.yml

```diff
@@ -71,12 +76,12 @@ jobs:
       SPARK_HOME: ${{ github.workspace }}
     steps:
     - name: Checkout Spark repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5 # v2
```

Comment:
Hm, I think we intentionally didn't pin this version because `actions` are considered as a safe one.

Reply:
Ok. Is there a specific reason why you think pinning a version would be an issue?

Reply:
I don't see a reason why we should necessarily pin the version since it will automatically leverage bug fixes.

Reply:
Ok. If you want me to revert I can do that. Let me know.

Reply:
Yeah, let's revert. This was actually already discussed when we pin other plugins IIRC.

Reply:
I have reverted the changes.